spec/fixtures/certificates/generate.sh

```diff
@@ -0,0 +1,10 @@
+# Warning: do not use the certificates produced by this tool in production. This is for testing purposes only
```

Comment:
This script has to be cited in the README.md with an explanation when to use it, so that next time the tests fails due to expired certificates there is a clear call to action on what to do.

Maybe a GH Action to update the certificates and create a PR would be event better.

Reply:
Fixed by 2a860b546d41141c04861abf47b8f4ea13b22e73